lib/modules/datasource/docker/index.ts

```diff
@@ -819,6 +820,36 @@ export class DockerDatasource extends Datasource {
     return digest;
   }
 
+  async getDockerHubTags(dockerRepository: string): Promise<string[] | null> {
+    if (!process.env.RENOVATE_X_DOCKER_HUB_TAGS) {
+      return null;
+    }
```

Comment:
We can remove this feature flagging once we've confirmed that this works at scale